modules/tinymce/src/themes/silver/main/ts/ui/dialog/IFrame.ts

```diff
@@ -34,14 +34,18 @@ const getDynamicSource = (initialData: Optional<string>, stream: boolean): IFram
             setSrcdocValue,
             (doc) => {
               const isElementScrollAtBottom = ({ scrollTop, scrollHeight, clientHeight }: HTMLElement) => scrollTop + clientHeight >= scrollHeight;
-              const isScrollAtBottom = isElementScrollAtBottom(doc.documentElement);
+              // TINY-10032: If documentElement is null, we assume document is empty and so scroll is at bottom.
+              const isScrollAtBottom = Optional.from(doc.documentElement).fold(Fun.always, isElementScrollAtBottom);
```

Comment:
```suggestion
              const isScrollAtBottom = Optional.from(doc.documentElement).filter(isElementScrollAtBottom).getOr(true);
```

Reply:
Can't we just use `.forall` like `Optional.from(doc.documentElement).forall(isElementScrollAtBottom)`?

Reply:
Ah yes `.forall` returns `true` for None values and so achieves the goal here. Implemented, thanks for the suggestions.

Reply:
ah I knew there was something nicer but couldn't remember. Thanks luke! I always forget `.exists` and `.forall`